configserver/pom.xml

```diff
@@ -184,6 +184,12 @@
       <version>${project.version}</version>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>com.yahoo.vespa</groupId>
+      <artifactId>jaxrs_client_utils</artifactId>
+      <version>${project.version}</version>
+      <scope>compile</scope> <!-- TODO Should ideally be provided, but this bundle is not installed as part of configserver. Orchestrator bundle has the same issue -->
```

Comment:
The orchestrator does NOT have the same issue (as jaxrs_client_utils, implied by "this bundle")